### ISSUE TITLE
visualize the feature map

### ISSUE THREAD

**Issue by immelo** [facebook/C3D#381]
Hello,
I just wanna visualize the deconvolution of C3D feature map of pre-trained models to explain what C3D have learned.I see the tool: extract_learned_filters.bin. can you explain how to use this in details?
Sincerely thank you

**Comment by dutran** [comment]
This tool is for extracting the learned filters, literately the weights of conv or fc layers in the net. If you'd like to do decovolution vis, more work needed, e.g. backprop the gradients all the way to the image space.
